README.md

```diff
@@ -239,6 +253,7 @@ Nim provides unique features for seamless and transparent interoperability with
 
 - [nimcr](https://github.com/PMunch/nimcr) - Running Nim code with Shebangs.
 - [nimr](https://github.com/Jeff-Ciesielski/nimr) - Run Nim programs like scripts.
+- [glob](https://github.com/citycide/glob) - Pure library for matching file paths against Unix style glob patterns.
```

Comment:
May be path handling?

Reply:
You mean as an own, new section?

Reply:
And yes, I also evaluated this to be part of the scripting section. 

What about combining the Scripting and the Processes sections together?

Reply:
> You mean as an own, new section?

Yeah, I think so.

Reply:
Done